outputs.tf

```
@@ -2,3 +2,8 @@ output "ipam" {
   description = "Object of all AWS VPC IPAM"
   value       = module.ipam
 }
+
+output "pools_cidr" {
```

Let's call it `pool_cidrs`, matching the input and the description...

```suggestion
output "pool_cidrs" {
```

---

updated, 

`===================================================== test session starts =====================================================
platform darwin -- Python 3.8.16, pytest-7.4.0, pluggy-1.0.0 -- /Users/johnricords/.pyenv/versions/3.8.16/bin/python3.8
cachedir: .pytest_cache
rootdir: /Volumes/Development
collected 6 items                                                                                                             

../tardigrade-ci/tests/terraform_pytest/test_terraform_install.py::test_modules[ipam_then_pool] PASSED                  [ 16%]
../tardigrade-ci/tests/terraform_pytest/test_terraform_install.py::test_modules[ipam_and_scope_then_pool] PASSED        [ 33%]
../tardigrade-ci/tests/terraform_pytest/test_terraform_install.py::test_modules[nested_pools] PASSED                    [ 50%]
../tardigrade-ci/tests/terraform_pytest/test_terraform_install.py::test_modules[ipam_and_scope_and_pool] PASSED         [ 66%]
../tardigrade-ci/tests/terraform_pytest/test_terraform_install.py::test_modules[ipam_scope_pool_cidr] PASSED            [ 83%]
../tardigrade-ci/tests/terraform_pytest/test_terraform_install.py::test_modules[ipam_and_pool_public] PASSED            [100%]

================================================ 6 passed in 400.19s (0:06:40) ================================================
[pytest//Volumes/Development/tardigrade-ci/tests/terraform_pytest]: Tests executed!`